playgrounds/remix/app/entry.server.tsx

```diff
@@ -0,0 +1,140 @@
+/**
+ * By default, Remix will handle generating the HTTP Response for you.
+ * You are free to delete this file if you'd like to, but if you ever want it revealed again, you can run `npx remix reveal` ✨
+ * For more information, see https://remix.run/file-conventions/entry.server
+ */
+
+import { PassThrough } from 'node:stream';
+
+import type { AppLoadContext, EntryContext } from '@remix-run/node';
+import { createReadableStreamFromReadable } from '@remix-run/node';
+import { RemixServer } from '@remix-run/react';
+import { isbot } from 'isbot';
+import { renderToPipeableStream } from 'react-dom/server';
+
+const ABORT_DELAY = 5_000;
+
+export default function handleRequest(
+  request: Request,
+  responseStatusCode: number,
+  responseHeaders: Headers,
+  remixContext: EntryContext,
+  // This is ignored so we can keep it in the template for visibility.  Feel
+  // free to delete this parameter in your app if you're not using it!
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  loadContext: AppLoadContext,
+) {
+  return isbot(request.headers.get('user-agent') || '')
```

Comment:
Since this is a simple playground where we don't care about bots, we can simplify this file to remove `handleBotRequest`. Basically `handleBrowserRequest` can be the default export.

Also `isbot` can be removed from dependencies in package.json (make sure to run `pnpm install` after that).

Reply:
Alright, will do.

Reply:
Done in latest [commit.](https://github.com/iTwin/iTwinUI/pull/1887/commits/d42b971de83f06b9700741ce183afc2e74e6e82e)

Reply:
When looking into https://github.com/iTwin/iTwinUI/pull/1887#discussion_r1511517993, I found that you can actually get rid of `entry.client` and `entry.server` files completely. Remix will automatically use the defaults. (This does mean we need to bring back `isbot`, but that's ok)

Reply:
Will do.